janusgraph-server tests fail in branch v0.5

When running `mvn test` from a fresh clone and branch `v0.5`, janusgraph-server tests fail
```
[INFO] ------------------< org.janusgraph:janusgraph-server >------------------
[INFO] Building janusgraph-server 0.5.4-SNAPSHOT                         [6/32]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:3.0.0-M2:enforce (enforce-dependency-convergence) @ janusgraph-server ---
[INFO]
[INFO] --- apache-rat-plugin:0.13:check (rat-checks) @ janusgraph-server ---
[INFO] Enabled default license matchers.
[INFO] Will parse SCM ignores for exclusions...
[INFO] Finished adding exclusions from SCM ignore files.
[INFO] 62 implicit excludes (use -debug for more details).
[INFO] 31 explicit excludes (use -debug for more details).
[INFO] 30 resources included (use -debug for more details)
[INFO] Rat check: Summary over all files. Unapproved: 0, unknown: 0, generated: 0, approved: 30 licenses.
[INFO]
[INFO] --- maven-resources-plugin:3.1.0:resources (default-resources) @ janusgraph-server ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO]
[INFO] --- maven-compiler-plugin:3.1:compile (default-compile) @ janusgraph-server ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-resources-plugin:3.1.0:testResources (default-testResources) @ janusgraph-server ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 2 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ janusgraph-server ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-surefire-plugin:2.22.1:test (default-test) @ janusgraph-server ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.janusgraph.graphdb.tinkerpop.JanusGraphChannelizerTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.451 s <<< FAILURE! - in org.janusgraph.graphdb.tinkerpop.JanusGraphChannelizerTest
[ERROR] bindingShouldExistAfterGraphIsCreated  Time elapsed: 0.394 s  <<< ERROR!
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
Caused by: java.lang.reflect.InvocationTargetException
Caused by: java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

Exception in thread "gremlin-server-shutdown" java.lang.NullPointerException
    at org.apache.tinkerpop.gremlin.server.GremlinServer.stop(GremlinServer.java:255)
    at org.apache.tinkerpop.gremlin.server.GremlinServer.lambda$new$0(GremlinServer.java:103)
    at java.lang.Thread.run(Thread.java:748)
[INFO] Running org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest
[ERROR] Tests run: 18, Failures: 0, Errors: 11, Skipped: 0, Time elapsed: 0.366 s <<< FAILURE! - in org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest
[ERROR] testSetupEmptyCredentialsGraphCreateUsernamePropertyKey  Time elapsed: 0.22 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupEmptyCredentialsGraphCreateDefaultUser  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupCredentialsGraphWithDefaultUserDontCreateUserAgain  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupEmptyCredentialsGraphCreateUsernameIndex  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testAuthenticateGenerateToken  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateGenerateToken(HMACAuthenticatorTest.java:87)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateGenerateToken(HMACAuthenticatorTest.java:87)

[ERROR] testAuthenticateBasicAuthInvalid  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateBasicAuthInvalid(HMACAuthenticatorTest.java:73)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateBasicAuthInvalid(HMACAuthenticatorTest.java:73)

[ERROR] testAuthenticateWithTimedOutToken  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithTimedOutToken(HMACAuthenticatorTest.java:152)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithTimedOutToken(HMACAuthenticatorTest.java:152)

[ERROR] testAuthenticateWithShortenedToken  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithShortenedToken(HMACAuthenticatorTest.java:115)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithShortenedToken(HMACAuthenticatorTest.java:115)

[ERROR] testAuthenticateWithBrokenToken  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithBrokenToken(HMACAuthenticatorTest.java:125)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithBrokenToken(HMACAuthenticatorTest.java:125)

[ERROR] testAuthenticateBasicAuthValid  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateBasicAuthValid(HMACAuthenticatorTest.java:59)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateBasicAuthValid(HMACAuthenticatorTest.java:59)

[ERROR] testAuthenticateWithToken  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithToken(HMACAuthenticatorTest.java:103)
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.createMockedAuthenticator(HMACAuthenticatorTest.java:177)
    at org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.HMACAuthenticatorTest.testAuthenticateWithToken(HMACAuthenticatorTest.java:103)

[INFO] Running org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.JanusGraphSimpleAuthenticatorTest
[ERROR] Tests run: 8, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.292 s <<< FAILURE! - in org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.JanusGraphSimpleAuthenticatorTest
[ERROR] testSetupEmptyCredentialsGraphCreateUsernamePropertyKey  Time elapsed: 0.213 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupEmptyCredentialsGraphCreateDefaultUser  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupCredentialsGraphWithDefaultUserDontCreateUserAgain  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupEmptyCredentialsGraphCreateUsernameIndex  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[INFO] Running org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.SaslAndHMACAuthenticatorTest
[ERROR] Tests run: 11, Failures: 0, Errors: 4, Skipped: 0, Time elapsed: 0.361 s <<< FAILURE! - in org.janusgraph.graphdb.tinkerpop.gremlin.server.auth.SaslAndHMACAuthenticatorTest
[ERROR] testSetupEmptyCredentialsGraphCreateUsernamePropertyKey  Time elapsed: 0.214 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupEmptyCredentialsGraphCreateDefaultUser  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupCredentialsGraphWithDefaultUserDontCreateUserAgain  Time elapsed: 0.001 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[ERROR] testSetupEmptyCredentialsGraphCreateUsernameIndex  Time elapsed: 0.002 s  <<< ERROR!
java.lang.IllegalArgumentException: Could not find implementation class: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager
Caused by: java.lang.ClassNotFoundException: org.janusgraph.diskstorage.inmemory.InMemoryStoreManager

[INFO] Running org.janusgraph.graphdb.tinkerpop.gremlin.server.handler.HttpHMACAuthenticationHandlerTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.233 s - in org.janusgraph.graphdb.tinkerpop.gremlin.server.handler.HttpHMACAuthenticationHandlerTest
[INFO] Running org.janusgraph.graphdb.tinkerpop.gremlin.server.handler.SaslAndHMACAuthenticationHandlerTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.202 s - in org.janusgraph.graphdb.tinkerpop.gremlin.server.handler.SaslAndHMACAuthenticationHandlerTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   JanusGraphChannelizerTest>AbstractGremlinServerIntegrationTest.setUp:67->AbstractGremlinServerIntegrationTest.startServer:99 » Runtime
[ERROR]   HMACAuthenticatorTest.testAuthenticateBasicAuthInvalid:73->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest.testAuthenticateBasicAuthValid:59->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest.testAuthenticateGenerateToken:87->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest.testAuthenticateWithBrokenToken:125->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest.testAuthenticateWithShortenedToken:115->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest.testAuthenticateWithTimedOutToken:152->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest.testAuthenticateWithToken:103->createMockedAuthenticator:177 » IllegalArgument
[ERROR]   HMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupCredentialsGraphWithDefaultUserDontCreateUserAgain:125 » IllegalArgument
[ERROR]   HMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateDefaultUser:80 » IllegalArgument
[ERROR]   HMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateUsernameIndex:107 » IllegalArgument
[ERROR]   HMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateUsernamePropertyKey:93 » IllegalArgument
[ERROR]   JanusGraphSimpleAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupCredentialsGraphWithDefaultUserDontCreateUserAgain:125 » IllegalArgument
[ERROR]   JanusGraphSimpleAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateDefaultUser:80 » IllegalArgument
[ERROR]   JanusGraphSimpleAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateUsernameIndex:107 » IllegalArgument
[ERROR]   JanusGraphSimpleAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateUsernamePropertyKey:93 » IllegalArgument
[ERROR]   SaslAndHMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupCredentialsGraphWithDefaultUserDontCreateUserAgain:125 » IllegalArgument
[ERROR]   SaslAndHMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateDefaultUser:80 » IllegalArgument
[ERROR]   SaslAndHMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateUsernameIndex:107 » IllegalArgument
[ERROR]   SaslAndHMACAuthenticatorTest>JanusGraphAbstractAuthenticatorTest.testSetupEmptyCredentialsGraphCreateUsernamePropertyKey:93 » IllegalArgument
[INFO]
[ERROR] Tests run: 45, Failures: 0, Errors: 20, Skipped: 0
```

Adding a dependency to janusgraph-inmemory (test scope), as was done [here](https://github.com/JanusGraph/janusgraph/pull/2155) for the master branch, solves the issue.

--- a/janusgraph-server/pom.xml
+++ b/janusgraph-server/pom.xml
@@ -38,5 +38,11 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.janusgraph</groupId>
+            <artifactId>janusgraph-inmemory</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 </project>

